src/ontology/mondo-edit.obo

```diff
@@ -574805,6 +574805,16 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-12T13:35:21Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7493" xsd:anyURI
 
+[Term]
+id: MONDO:1040011
+name: vertebral artery dissection
+def: "The result of a tear in one of the major cervicocerebral arteries, allowing blood to enter the wall of the artery and split its layers, resulting in either stenosis or aneurysmal dilatation of the vessel." [PMID:19096659, PMID:28722857]
```

Comment:
let's discuss the definition to make sure it is clear that this term represents a disease/disorder